### ISSUE TITLE
Broken output with NotoSansMono Regular

### ISSUE THREAD

**Issue by nyuichi** [gfngfn/SATySFi#155]
# TL;DR

<img width="841" alt="2019-02-15 18 54 01" src="https://user-images.githubusercontent.com/540144/52848841-11ab5b00-3153-11e9-893c-d01a1a1c29b3.png">

# How to reproduce

I uploaded a set of sources I used to test this as a single gist repo: https://gist.github.com/nyuichi/d0e83c74a6cfeb40835f7c37a46e285f
Terminal output from satysfi is also attached to the gist.
The version of satysfi I used was 4d1e62d.

# Notes

According to @gfngfn san,

- copy&pasting from the broken pdf is working correctly
- Opening it with Acrobat, gfn san got warnings that the font (i.e. NotoSansMono Regular) was not properly embedded in the pdf.

**Comment by matsud224** [comment]
This bug seems to be fixed by gfngfn/otfm#5.

(Side note: Unhinted version of NotoSansMono causes this problem but hinted version does not.)
